docs/src/components/pricing/WhatToExpect.tsx

```diff
@@ -1,114 +1,66 @@
 
 export default function WhatToExpect() {
   return (
-    <Container sx={{ py: { xs: 4, sm: 6, md: 8 } }}>
-      <Typography
-        variant="h2"
-        sx={{ maxWidth: { xs: 280, sm: 320, md: 400 }, mb: { xs: 2, sm: 4 } }}
-      >
-        What to expect from our available plans
-      </Typography>
-      <Grid container spacing={{ xs: 2, sm: 4 }}>
-        <Grid item xs={12} sm={6}>
-          <Paper variant="outlined" sx={{ p: 2, height: '100%' }}>
-            <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
-              <LocalOfferOutlinedIcon fontSize="small" color="primary" />
-              <Typography
-                fontWeight="bold"
-                component="h3"
-                color="text.primary"
-                variant="body2"
-                sx={{ ml: 1 }}
-              >
-                Volume discount
-              </Typography>
-            </Box>
-            <Typography variant="body2" color="text.secondary">
-              The licenses are on a per-developer basis. We offer the following tiered discounts
-              from list prices when purchasing more than one license for your development team:
-            </Typography>
-            <Box component="ul" sx={{ px: 2.5, typography: 'body2', color: 'text.secondary' }}>
-              <li>2-5 Licenses: 10% discount</li>
-              <li>6-10 Licenses: 15% discount</li>
-              <li>11+: License capped, extra developers do not need to be licensed.</li>
-            </Box>
-          </Paper>
-        </Grid>
-        <Grid item xs={12} sm={6}>
-          <Paper variant="outlined" sx={{ p: 2, height: '100%' }}>
-            <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
-              <AllInclusiveOutlinedIcon fontSize="small" color="primary" />
-              <Typography
-                fontWeight="bold"
-                component="h3"
-                color="text.primary"
-                variant="body2"
-                sx={{ ml: 1 }}
-              >
-                Perpetual license
-              </Typography>
-            </Box>
-            <Typography variant="body2" color="text.secondary">
-              With your purchase, you are granted a license to use a version of the product in
-              perpetuity. There are no further charges unless you choose to renew support and
-              maintenance to cover newer versions. Please note that while the use of the software is
-              perpetual, support and maintenance are not. We roll bug fixes, performance
-              enhancements, and other improvements into new releases; we don&apos;t patch, fix, or
-              in any way alter older versions.
-            </Typography>
-          </Paper>
+    <Container sx={{ py: { xs: 4, md: 8 } }}>
+      <Grid container spacing={{ xs: 2, md: 3 }} columns={{ xs: 4, md: 12 }}>
+        <Grid item xs={6}>
+          <Typography variant="h2">What to expect from our available plans</Typography>
```

Comment:
This layout feels broken:

<img width="1199" alt="Screenshot 2022-05-12 at 02 18 44" src="https://user-images.githubusercontent.com/3165635/167967749-93e906ae-1e2d-4a59-8eff-1e852a9f9ab7.png">

Would this one feel more natural?

<img width="1218" alt="Screenshot 2022-05-12 at 02 19 36" src="https://user-images.githubusercontent.com/3165635/167967805-8c913be6-364b-4eb0-982b-be89a3e860ae.png">

Reply:
The second option was how it was previously 😅 What bothered me to change it and try a different layout was the extra amount of white space that the box on the right gained due to having less content than the one from the left. It's interesting the "feels broken" though, I wonder what exactly causes this sensation

Reply:
This would be fixed with https://github.com/mui/material-ui/pull/32458#discussion_r871447189.